easybuild/easyconfigs/m/Mesa/Mesa-10.5.5-intel-2015a-Python-2.7.10.eb

```diff
@@ -69,6 +68,6 @@ sanity_check_paths = {
     'dirs': []
 }
 
-maxparallel = 1
+#maxparallel = 1
```

Comment:
ok, then drop this line, I can test on a 16 and 24-core system, if it works there, it should be fine